packages/fxa-content-server/app/tests/spec/models/reliers/relier.js

```diff
@@ -110,10 +110,6 @@ describe('models/reliers/relier', function() {
     });
   });
 
-  ['trailhead-1', 'chromeless'].forEach(value => {
```

Comment:
@apsychogirl could you please just remove the `chromeless`, let's keep the trailhead-1 while we still support the `style` query param.

Reply:
> @apsychogirl could you please just remove the `chromeless`, let's keep the trailhead-1 while we still support the `style` query param.

Should I also remove the `value` query param which was earlier taking the value for `Chromeless` in line 114?